integration-tests/yarn.lock

```diff
@@ -2116,7 +2116,7 @@ create-require@^1.1.0:
 
 "creditcoin-js@file:../creditcoin-js/creditcoin-js-v0.9.0.tgz":
   version "0.9.0"
-  resolved "file:../creditcoin-js/creditcoin-js-v0.9.0.tgz#274e28b37ef15f800c5bfc2d1763bcd6ee953078"
+  resolved "file:../creditcoin-js/creditcoin-js-v0.9.0.tgz#87991bac088cb9c6763c9bbc44b8199892ad6db3"
```

Comment:
Can you also update `scripts/js/yarn.lock` ? 

Both directories depend on the creditcoin-js tarball and we try keeping the lock files in sync as much as possible.

Reply:
Hmm, what is this a result of?

Why would the checksum change?

Reply:
Probably you had called `yarn build` locally and maybe afterwards updated some dependencies ?

Reply:
Is there something in CI we can add to catch this or automatically update this for us?

Reply:
There are no checks for this at present but I imagine a simple script under `./scripts/` / sanity-tests could do the job.